docs/markup-language/README_es.md

```diff
@@ -0,0 +1,1618 @@
+'''
+
+otra disposición
+```
+
+La anotación de notas se puede colapsar dentro del cuerpo del modelo. Se puede usar para añadir notas notas a lo largo de ello para aclarar o comentar sobre el contenido. La sintáxis para la anotación de notas es lo siguiente:
+
+```
+una disposición
+
+"""
+Esto es una anotación de membrete para explicar la disposicón, y por qué viene bien tenerlo aquí.
+"""
+
+otra disposición
+```
+
+## Tablas
+
+Los datos tabulares se muestran fácilmente usando tablas. Una tabla se define con el uso de carácteres de pleca (`|`) como separadores de columna, y rayas (`-`) para separar el membrete de la tabla de las filas de datos.
+
+```
+| Membrete 1 | Membrete 2 | Membrete 3 |
+| ---------- | ---------- | ---------- |
+| Datos 1    | Datos 2    | Datos 3    |
+| {{var1}}   | {{var2}}   | {{var3}}   |
+```
+
+Observa que los separadores de columna no tienen que alinearse de fila a fila, así que lo siguiente también es una definición válida para una tabla.
+
+```
+| H1 | Membrete 2 | H3 |
+| - | - | - |
+| Datos 1 | D2 | D3 |
+```
+
+Las células de una tabla pueden contener contenido en texto, variables, o condicionales.
+
+## Condicionales y Ramos de Decisión
+
+Puedes poner en marcado un acuerdo con "condicionales" para incrustar lógica avanzada en un acuerdo, lo cual le ayudará a crear acuerdos legales más dinámicos y personalizables.
+
+### Condicionales Básicas
+
+Actualmente, las condicionales crean preguntas de "si" o "no" en nuestra aplicación de generación de formularios. Cuando un usuario entra un "si", la condicional producirá texto, variables, llamadas a un contrato inteligente, y/o invocará otra condicional. La construcción básica deuna condicional es lo siguiente:
+
+```
+{{Nombre de la Condicional "Pregunta que se hace al Usuario" =>
+Texto que se incluirá en un acuerdo si el usuario elige 'si'}}
+```
+
+Observa que hay requisitos firmes que se han de cumplir al crear una condicional. Primeramente, tiene que iniciar a la condicional con `{{` y acabarla con `}}`. Por segundo, tiene que nombrarle a la condiional e incluir algo escrito entre comillas que sirve como impulso para el usuario.
+
+### Condicionales Anidadas
+
+Las condicionales también se pueden agrupar para crear a un árbol de decisiones. Dicho de otro modo:
+
+```
+{{ Nombre de la Condicional "Pregunta que se hace al Usuario" =>
+  Texto que se incluirá en un acuerdo si el usuario elige 'si'
+  {{Sub-Condicional-1 "Texto de Sub-Pregunta 1" => Texto}}
+  {{Sub-Condicional-2 "Texto de Sub-Pregunta 2" => Texto}}
+  {{Sub-Condicional-3 "Texto de Sub-Pregunta 3" => Texto}}
+}}
+```
+
+Para verle a esto en acción, tome como ejemplo el lenguaje estándar siguiente que se encuentra al principio de un acuerdo mutuo de confidencialidad estándar :
+
+```
+Este Acuerdo Mutuo de Confidencialidad  (este "Acuerdo") se hace a partir de
+[[Fecha de Vigencia: Fecha]], por y entre [[ParteA]] ("[[ParteA Abreviatura]]")
```

Comment:
Do you think it would be clear to the reader that they still have to use the English-language term for the variable (I don't think `Fecha` will compile?) I think the way above was a bit clearer (For example - `nombre: Text;`)

Reply:
Thank you, this is one of those instances that I mentioned where I had a hard time tracking what was a variable name and what was a semantic natural language term. I'll correct this.

Reply:
I think there may have been other examples of this in the text as well. Do you want me to flag them directly?

Reply:
No need. I went through looking at the various variable input types (address, date, DateTime, etc) and corrected the ones that were translated when they shouldn't have been -- mainly 'Date'. Just pushed up.